lib/models/living.py

```diff
@@ -0,0 +1,48 @@
+from dataclasses import dataclass
+from typing import List, Dict
+
+from lib.models.character_class import CharacterClass
+from lib.models.entity import *
+from lib.models.enums import Ability, Skill, Alignment
+
+
+@dataclass
+class Living(Entity):
+    name: str
+    description: str
+    character_class: CharacterClass 
```

Comment:
Later one we could probably inherit this, or somehow otherwise use it as a basis for a `PlayerFactory` or something